tasks.py

```diff
@@ -25,7 +25,7 @@ def is_truthy(arg):
     {
         "nautobot_golden_config": {
             "nautobot_ver": "1.0.1",
-            "project_name": "nautobot-golden-config",
+            "project_name": "nautobot_golden_config",
```

Comment:
@ubajze is this fine?

Reply:
it is only used here:
```
compose_command = f'docker-compose --project-name {context.nautobot_golden_config.project_name} --project-directory "{context.nautobot_golden_config.compose_dir}"'
```
So it should be fine in either case.

Reply:
Cookiecutter uses `_`

https://github.com/networktocode-llc/cookiecutter-ntc/blob/04b841285d81a03d18015a26aeb4271c5f23d154/nautobot-plugin/%7B%7B%20cookiecutter.project_slug%20%7D%7D/tasks.py#L42